vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -129,6 +129,19 @@ winrt::IInspectable DynamicAutomationPeer::GetPatternCore(winrt::PatternInterfac
       (HasAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityStates::Expanded) ||
        HasAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityStates::Collapsed))) {
     return *this;
+  } else if (patternInterface == winrt::PatternInterface::Value && 
+      // TODO ?restrict to some control types?
+      HasAccessibilityValue(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityValue::Text)) {
```

Comment:
is this right? e.g. what if it's a non-text control e.g. a combobox, or a numeric control like slider?

Reply:
Slider is handled by the Range pattern. Handling ComboBox for the Value pattern. 
+question in comment above